core/src/components/card/card.tsx

```diff
@@ -1,29 +1,34 @@
-import { Component, h, Prop, Event, EventEmitter, Element, State, Host } from '@stencil/core';
-
+import { Component, h, Prop, Event, EventEmitter, Element, Host } from '@stencil/core';
+import { hasSlot } from '../../utils/utils';
+
+/**
+ * @slto header - Slot for the Card header.
```

Comment:
```suggestion
 * @slot header - Slot for the Card header.
```

still typo, did you forget to push?

Reply:
So weird.. thought I commited a change for this...